sema/commons/glob/globbery.py

```
@@ -0,0 +1,125 @@
+    onlyFiles: bool = False
+) -> List[Path]:
+    """
+    Get all paths under `root` that match any of the globs in `includes`
+    and none of the globs in `excludes`.
+    @param root: The root path to search under.
+    @param includes: A list of globs to include.
+    @param excludes: A list of globs to exclude.
+    @param onlyFiles: If True, only return files, not directories.
+    @return: A list of paths that match the globs.
+    """
+    found: Set[Path] = set()
```

Ruff (0.8.2)

Use `set` instead of `Set` for type annotation

Replace with `set`

(UP006)